modules/timepoint_list/php/timepoint_list.class.inc

```diff
@@ -84,14 +85,11 @@ class Timepoint_List extends \NDB_Menu
         ServerRequestInterface $request,
         RequestHandlerInterface $handler
     ) : ResponseInterface {
-        $attribute = $request->getAttribute("CandID");
-        if ($attribute !== null) {
-            // FIXME: Update the code to add the singleton here, not the identifier.
-            $this->candID = $attribute;
-        } else {
-            $gets         = $request->getQueryParams();
-            $this->candID = $gets['candID'];
-        }
+        // FIXME: Update the code to add the singleton here, not the identifier.
+        $this->candID = new CandID(
+            $request->getAttribute('CandID')
+            ?? $request->getAttribute('candID')
```

Comment:
shouldn't this be `?? $request->getQueryParams()['candID']` ??

Reply:
or better: fix the place where the capitalization is wrong?